inclusive-design/wecount.inclusivedesign.ca
# Issue-91: Add dynamic routing to share one single template

## Description
Fixes #91: these routes: /about/, /tools/, /inclusion-challenges/, /our-data/ now shares one single template at `pages/_requestedPath/index.vue`.

## Steps to test

1. Run `npm run dev` to start the website locally at `http://localhost:3000`;
2. Click through links in the navigation section to verify each page.

**Expected behavior:**
All pages behave the same as the master branch.
